tardis/io/decay.py

```diff
@@ -86,4 +86,34 @@ def decay(self, t):
         
         df = IsotopeAbundances.from_materials(decayed_materials)
         df.sort_index(inplace=True)
-        return df 
\ No newline at end of file
+        return df 
+
+    def as_atomic_numbers(self, abundance, t, normalize=True):
+        """
+        Merge decayed Isotope dataframe with abundance passed as parameter 
+
+        Parameters
+        ----------
+
+        t: float or astropy.units.Quantity
+            If float it will be understood as days
+        abundance: pd.DataFrame 
+        normalize : bool
+            If true, resultant dataframe will be normalized along the coloumn
+
+        Returns:
+            : merged abundances
+        """
+        #Drop mass_number coloumn in isotopic_abundance dataframe
+        isotope_abundance = self.decay(t).reset_index(
+            level='mass_number').drop('mass_number', axis=1)
+
+        #Merge abundance dataframes
+        modified_df = pd.concat([isotope_abundance, abundance])
```

Comment:
@wkerzendorf  For `pd.add()` , `index values` for both `Dataframe` must be equal, then only result is correct, otherwise, index values not present in one dataframe , are filled with `NaN` values